department-of-veterans-affairs/abd-vro
# Mcp 1887 populate assessment result

## What was the problem?


Associated tickets or Slack threads:

- [MCP-1887](https://amida.atlassian.net/browse/MCP-1887)

## How does this fix it?[^1]
After claim is processed in full-health-assessment the assessment result is populated in the db

## How to test this PR
- try out full-health-assessment
- check database to see that assessment result is populated and evidence count summary holds proper JSON


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
